src/Agent.Worker/Agent.Worker.csproj

```diff
@@ -13,8 +13,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="vss-api-netcore" Version="0.5.61-private" />
+    <PackageReference Include="vss-api-netcore" Version="0.5.newversion-private" />
```

Comment:
Check with Ting. Does this prevent us from consuming a new web API unless we produce it from your feature branch? I would expect server side to be in master.